covidtask/covidtask/models.py

```diff
@@ -1,21 +1,29 @@
 from django.db import models
 from django.contrib.auth.models import User
 
+
 class Country(models.Model):
-    name = models.CharField(max_length=255)
-    confirmed = models.IntegerField(blank=True, null = True)
-    death = models.IntegerField(blank=True, null = True)
-    recoverd = models.IntegerField(blank=True, null = True)
+    name = models.CharField(max_length=255, unique=True)
     updated_at = models.DateTimeField(auto_now=True)
-    subscription = models.ManyToManyField(User,related_name ="subscriptions",db_table="Subscription",verbose_name="subscriptions",blank=True,null = True)
+    subscription = models.ManyToManyField(User, related_name="subscriptions", db_table="Subscription",
+                                          verbose_name="subscriptions", blank=True, null=True)
+
     def __str__(self):
         return self.name
 
-class CountriesHistory(models.Model):
-    country_id = models.ForeignKey(Country,on_delete=models.CASCADE)
-    confirmed = models.IntegerField(blank=True, null = True)
-    death = models.IntegerField(blank=True, null = True)
-    recoverd = models.IntegerField(blank=True, null = True)
-    created_at = models.DateTimeField(auto_now=True)
+
+class CountryDailyCases(models.Model):
+    country_id = models.ForeignKey(Country, on_delete=models.CASCADE, related_name="country_daily_cases")
+    confirmed = models.IntegerField(blank=True, null=True)
+    death = models.IntegerField(blank=True, null=True)
+    recoverd = models.IntegerField(blank=True, null=True)
+    confirmed_agg = models.IntegerField(blank=True, null=True)
+    death_agg = models.IntegerField(blank=True, null=True)
+    recoverd_agg = models.IntegerField(blank=True, null=True)
```

Comment:
storing aggregated data like this will make it much harder to maintain data which can cause severe data inconsistencies.